test/parallel_testsuite.py

```diff
@@ -150,6 +155,10 @@ def combine_results(self, result, buffered_results):
     results = sorted(buffered_results, key=lambda res: str(res.test))
     result.core_time = 0
 
+    # shared data structures are hard in the python multi-processing world, so
+    # use a file to share the flaky test information across test processes.
+    flaky_tests = open(common.flaky_tests_log_filename).read().split() if os.path.isfile(common.flaky_tests_log_filename) else []
```

Comment:
I think `.splitlines()` would make sense here.

Reply:
Actually you can just do `.readlines()` in a single call.

Reply:
`readlines()` is supposed to work with text files so I'm somewhat confused but this... but not a big deal I guess

Reply:
![Uploading image.png…]()

Reply:
<img width="831" height="228" alt="image" src="https://github.com/user-attachments/assets/2ca78ec9-5c50-4d1b-bade-4f287bdcdfbc" />

Reply:
Well, that's just worse than `open().read().split()` :/